credential/store/src/test/java/org/wildfly/security/credential/store/impl/KeyStoreCredentialStoreTest.java

```diff
@@ -221,7 +225,19 @@ public void symbolicLinkLocation() throws Exception {
         final KeyStoreCredentialStore retrievalStore = new KeyStoreCredentialStore();
 
         final File symbolicLinkFile = new File(tmp.getRoot(), "link");
-        Files.createSymbolicLink(Paths.get(symbolicLinkFile.getAbsolutePath()), Paths.get(keyStoreFile.getAbsolutePath()));
+        try {
+            Files.createSymbolicLink(Paths.get(symbolicLinkFile.getAbsolutePath()), Paths.get(keyStoreFile.getAbsolutePath()));
```

Comment:
Thanks for working on this @cam-rod!

To actually skip the test when not running in administrator mode on Windows, we could use `assumeTrue` instead, e.g., see https://github.com/wildfly-security/wildfly-elytron/blob/b7319d195ed146903aac6caacf8574048a92eb8d/tests/base/src/test/java/org/wildfly/security/ssl/TLS13AuthenticationTest.java#L83-L84

For our case, we could create a helper method that calls `createSymbolicLink` and returns true if it could be created and false otherwise. We can this use a call to this method in our `assertTrue` statement.

Let me know if you have any questions.

Reply:
Alright, updated.